Forge.TreeWalker/src/TreeWalkerSession.cs

```diff
@@ -56,6 +54,18 @@ public class TreeWalkerSession : ITreeSession
         /// </summary>
         public static string IntermediatesSuffix = "_Int";
 
+        /// <summary>
+        /// The TreeInput suffix appended to the end of the key in forgeState that maps to this tree walking session's TreeInput object.
+        /// </summary>
+        public static string TreeInputSuffix = "TI";
```

Comment:
"TI" => "_TI" to follow the pattern used on other Suffix?

Reply:
The tree-level persisted rows do not have "_" because the final key looks like this:
<RootSessionId>_<SessionId>_**TI**

Action-level persisted rows have the "_" because the final key looks like this:
<RootSessionId>_<SessionId>_<TreeActionKey>**_AR**